tests/drivers/can/host/README.rst

```diff
@@ -107,7 +114,7 @@ Twister. Below is an example for running on the :ref:`lpcxpresso55s36`:
 
 .. code-block:: shell
 
-   west twister -v -p lpcxpresso55s36/lpc55s36 --device-testing --device-serial /dev/ttyACM0 -X can -T tests/drivers/can/host/ --pytest-args=--can-context=can0
+   west twister -v -p lpcxpresso55s36/lpc55s36 --device-testing --device-serial /dev/ttyACM0 -X can:can0 -T tests/drivers/can/host/
```

Comment:
for a CI round, we usually use a map file to host all supported fixtures, using commadline for one fixture is ok, but in board CI we will need a easy way to handle this. how about adding the ":" support to mapfile parse as well?

Reply:
It's already there? That's the whole point of this PR.

Reply:
so event I define the fixture as below in the hardware map file, this will works right?
```
  fixture:
  - can:can0 
```

Reply:
Correct.